### ISSUE TITLE
Forecasting airline passengers using designer machine learning - Alexander Backus, Jan van der Vegt

### ISSUE THREAD

**Issue by trfore** [numfocus/YouTubeVideoTimestamps#6]
https://www.youtube.com/watch?v=A-azJ8qmwug

0:00 Introduction
0:50 Problem: Predicting Passenger Number & Use Cases
1:38 Problem: Unique Forecasting Constraint - Shrinking Horizon
2:08 System Requirements
2:50 System Design
3:23 "Designer Machine Learning" Definition
3:56 Data: Artificial Flight-bookings
5:07 Data: Features
6:24 Model: Simple Linear Model & ANN
7:15 Model: Feed-Forward Deep Neural Network
7:48 Model: Loss Function - MSE
8:25 Keras Code Example
9:33 Use Case: Aircraft Allocation
10:25 Evaluation: Probability of Capacity Overflow 
11:45 Model: Conditional Density Estimation
13:48 Model: Updated ANN Outputs (Mu & Sigma) & Loss Function
15:04 Keras Code Example for Conditional Density Estimation
16:46 Model: Mixture Density
17:09 Model: Mixture Density Networks
17:51 Challenges: Selecting Distributions & Numerical Optimization
19:06 Sequence Feature Extraction
19:41 Model: Representational Learning & Recurrent Neural Network
21:08 Keras Code Example for RNN with LSTM
21:57 Challenges: Non-uniform Time Deltas & Flight Dependencies
23:08 Key Take-aways
24:32 Q&A: Q1
26:04 Q&A: Q2 
27:26 Q&A: Q3
29:16 Q&A: Q4
30:58 Q&A: Q5
32:40 Q&A: Q6

**Comment by trfore** [comment]
Also the slides hyperlink is broken, with no clear replacement available at https://pydata.org/amsterdam2018/schedule/presentation/8/

**Comment by logankilpatrick** [comment]
Thank you!!
